nifi-docker/dockermaven/Dockerfile

```diff
@@ -69,7 +69,9 @@ ENV NIFI_LOG_DIR=${NIFI_HOME}/logs
 RUN groupadd -g ${GID} nifi || groupmod -n nifi `getent group ${GID} | cut -d: -f1` \
     && useradd --shell /bin/bash -u ${UID} -g ${GID} -m nifi \
     && apt-get update \
-    && apt-get install -y jq xmlstarlet procps
+    && apt-get install -y jq xmlstarlet procps \
+    && apt-get clean  \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
This step removes the apt lists which is an unnecessary step that could cause confusion for people who extend the docker image.